server/test/services/export/AbstractExporterTest.java

```diff
@@ -76,11 +77,13 @@ public abstract class AbstractExporterTest extends ResetPostgres {
   protected ApplicationModel applicationFive;
   protected ApplicationModel applicationSix;
   protected ApplicationModel applicationSeven;
+  private ApplicationStatusesRepository apprepo;
 
   @Before
   public void setup() {
     programAdminApplicationService = instanceOf(ProgramAdminApplicationService.class);
     programService = instanceOf(ProgramService.class);
+    apprepo = instanceOf(ApplicationStatusesRepository.class);
```

Comment:
nitpick (non-blocking): rename this var `appRepo` or `appStatusRepo` for readability

Reply:
done!